extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationServiceSpec.js

```diff
@@ -165,4 +176,22 @@ describe('ItemSelectionInputValidationService', function() {
       message: 'Please ensure the choices are unique.'
     }]);
   });
+
+  it(
+    'should expect more that 1 element to be  in the rule input, if the ' +
+    '"proper subset" rule is used.',
+    function() {
+      // Modify values of customization arguments to get warning.
+      customizationArguments.minAllowableSelectionCount.value = 0;
```

Comment:
Why does the proper subset check depend on minAllowableSelectionCount? I thought the check was for the validity of the rule, not the student's answer.

Reply:
It is not needed (I kept it before, did not remove it  (I thought it will trigger other tests before, removed it now).